### PR TITLE
[6/n][resources ui] Highlight assets which use resources in resources UI

### PR DESCRIPTION
**Pull request by benpankow**
## Summary

Lists and links to assets that depend on resources in the resource's page in the Dagster UI.

Screen Shot 2023-03-17 at 12 15 31 PM

## Test Plan

tested locally

### CODE REVIEW

**Comment by benpankow** [comment]
Current dependencies on/for this PR:
* master
  * **PR #12728** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12728" target="_blank">Graphite</a> 
    * **PR #12729** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12729" target="_blank">Graphite</a> 
      * **PR #12832** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12832" target="_blank">Graphite</a> 
        * **PR #12833** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12833" target="_blank">Graphite</a> 
          * **PR #13007** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13007" target="_blank">Graphite</a> 
            * **PR #13008** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13008" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13008?utm_source=stack-comment).

File: js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx [hunk 1]
```diff
@@ -346,41 +345,89 @@ const ResourceConfig: React.FC<{
...
+          <SectionHeader>
+            <Subheading>Assets</Subheading>
+          </SectionHeader>
+          <Table>
+            <thead>
+              <tr>
+                <th>Asset key</th>
+              </tr>
+            </thead>
+            <tbody>
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx:417]: Same question as above

**Review by hellendag — APPROVED** [review]

**Comment by benpankow** [comment]
@benpankow queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13008).

**Comment by benpankow** [comment]
[Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13008) rebased this pull request as part of a merge.

**Comment by benpankow** [comment]
@benpankow merged this pull request with [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13008).

**Pull request merged**